django/pyrog/api/serializers/import_export.py

```diff
@@ -0,0 +1,265 @@
+"""Serializer for the full mapping.
+
+These serializers add their child nested representations. Parent relation will be
+deduced from the hierarchy. Some relations (to owners and columns) use an exported
+primary key to make reference from other branch in the hierarchy.
+
+    Usage example:
+
+    from pyrog.api.serializers.import_export import SourceSerializer
+"""
+
+from typing import Mapping
+
+from rest_framework import serializers
+
+from pyrog.models import (
+    Attribute,
+    Column,
+    Condition,
+    Credential,
+    Filter,
+    Input,
+    InputGroup,
+    Join,
+    Owner,
+    Resource,
+    Source,
+)
+
+
+class _ColumnField(serializers.PrimaryKeyRelatedField):
```

Comment:
can you document this a bit more and explain why we need to validate the data when importing a mapping ?